CONTRIBUTING.md

```diff
@@ -28,7 +30,7 @@ For example, you can simply activate this environment and start using Snorkel:
 ```bash
 source .env/bin/activate
 
-python3 -c "import snorkel; print(dir(snorkel.labeling))"
+python3 -c "import snorkel.labeling; print(dir(snorkel.labeling))"
```

Comment:
looks good, but why not just `import snorkel; print(dir(snorkel))`?

Reply:
won't be informative